src/go/rpk/pkg/kafka/client.go

```diff
@@ -47,8 +49,15 @@ func DefaultConfig() *sarama.Config {
 
 // Overrides the default config with the redpanda config values, such as TLS.
 func LoadConfig(conf *config.Config) (*sarama.Config, error) {
+	var err error
```

Comment:
nit: this seems unnecessary, given that the value for `c` isn't determined by different code branches.

Reply:
Ok removed.